src/js/components/ServicesTable.js

```diff
@@ -74,10 +153,135 @@ var ServicesTable = React.createClass({
     );
   },
 
+  onAcceptDestroyConfirmDialog: function () {
+    let service = this.state.serviceToChange;
+    let isGroup = service instanceof ServiceTree;
+
+    this.setState({disabledDialog: ServiceActionItem.DESTROY}, () => {
+      let serviceID = service.getId();
+
+      if (isGroup) {
+        MarathonStore.deleteGroup(serviceID);
+      } else {
+        MarathonStore.deleteService(serviceID);
+      }
+    });
+  },
+
+  onAcceptSuspendConfirmDialog: function () {
+    let service = this.state.serviceToChange;
+    let isGroup = service instanceof ServiceTree;
+
+    this.setState({disabledDialog: ServiceActionItem.SUSPEND}, () => {
+      let serviceID = service.getId();
+
+      if (isGroup) {
+        MarathonStore.editGroup({
+          id: serviceID,
+          scaleBy: 0
+        });
+      } else {
+        MarathonStore.editService({
+          id: serviceID,
+          instances: 0
+        });
+      }
+    });
+  },
+
   onMarathonAppsChange: function () {
     this.forceUpdate();
   },
 
+  onActionsItemSelection: function (service, actionItem) {
+    this.setState({
+      serviceToChange: service,
+      serviceActionDialog: actionItem.id
+    });
+  },
+
+  closeDialog: function () {
+    this.setState({
+      disabledDialog: null,
+      errorMsg: null,
+      serviceActionDialog: null,
+      serviceToChange: null
+    });
+  },
+
+  getErrorMessage: function () {
+    let {errorMsg} = this.state;
+    if (!errorMsg) {
+      return null;
+    }
+    return (
+      <p className="text-danger flush-top">{errorMsg}</p>
+    );
+  },
+
+  getDestroyConfirmDialog: function () {
+    const {state} = this;
+    let service = state.serviceToChange;
+
+    let message = (
+      <div className="container-pod flush-top container-pod-short-bottom">
+        <h2 className="text-danger text-align-center flush-top">Destroy Service</h2>
```

Comment:
We need to check if this is a group (`service instanceof ServiceTreee` should do) and ask to `Destroy Group` in that case.